### PR TITLE
Github buttons

### PR DESCRIPTION
**Pull request by spacephoton**
* about page
image

* left footer
image

Could do some prettifications but we can do that later. 

#34

### CODE REVIEW

**Comment by spacephoton** [comment]
We could consider putting it on the website header but that might be overkill

**Comment by chaoran-chen** [comment]
Hi @TKGZ! Thanks for your work and I'm very sorry for not being clearer but I really only thought about including the small Github mark/icon (see below) and link it to our repository but having buttons for starring + watching seems too much for me. The purpose of the mark is to let everyone know that it is open source and give them a place to submit issues but I don't consider stars as very important.

image

**Comment by spacephoton** [comment]
No worries this only took a few minutes, where do you think it should be put?

**Comment by chaoran-chen** [comment]
hmm. top-right? I'm really not sure..

**Comment by spacephoton** [comment]
@chaoran-chen I think it is okay to put it in the about page, and nowhere else: 
image

**Comment by spacephoton** [comment]
How about just keeping the small issue icon button, and removing the watch/star?

**Comment by spacephoton** [comment]
image

**Comment by spacephoton** [comment]
and maybe replace it with the github icon? 
image

**Comment by chaoran-chen** [comment]
I think that's still too much. In my opinion, a simple mark is entirely sufficient.

**Comment by spacephoton** [comment]
image
Small button only

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "@types/react-bootstrap": "^0.32.25",
     "@types/react-csv": "^1.1.1",
     "@types/react-dom": "^17.0.0",
+    "@types/react-github-button": "^0.1.0",
```
Comment (chaoran-chen) [inline package.json]: I guess that we won't need this then?

**Review by chaoran-chen — CHANGES_REQUESTED** [review]
Thanks!

Please just remove the unused dependency. You can then merge :)

**Pull request merged**